aerogear/aerogear.org
# AGDIGGER-205 digger java client specs

https://issues.jboss.org/browse/AGDIGGER-205

2 commits:
1. Copy paste the HTML generated by Maven Javadoc plugin
2. Create links

Verification:
* Start up the website locally, using the instructions in the readme file
* Open http://127.0.0.1:4000/docs/specs/ and see the link there
* Open http://127.0.0.1:4000/digger/ and see the link there
